test/dogear.test.js

```diff
@@ -105,7 +102,7 @@ describe('Plugin reports on paths.', (tap) => {
   });
 
 
-  tap.test('Reports stats for request.', (t) => {
+  tap.test('Reports stats for request.', async t => {
```

Comment:
For clarity of code, I prefer that all arrow functions have parenthesis around their parameters. Please change these to `async (t) =>`.

Reply:
I'll change. But could you please adjust your linter rules if like a certain style more than another?